compiler/rustc_hir/src/hir.rs

```diff
@@ -658,6 +657,11 @@ pub struct Crate<'hir> {
 }
 
 impl Crate<'hir> {
+    pub fn module(&self) -> &'hir Mod<'hir> {
+        let i = self.owners[CRATE_DEF_ID].as_ref().unwrap();
+        if let OwnerNode::Crate(m) = i { m } else { panic!() }
```

Comment:
```suggestion
        if let Some(OwnerNode::Crate(m)) = i { self.owners[CRATE_DEF_ID].as_ref() } else { unreachable!() }
```
Nit: one panic should be enough.